app/src/main/java/com/example/hulk/mtindo/Createstore.java

```diff
@@ -53,6 +59,9 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_createstore);
+        Firebase.setAndroidContext(this);
+        final Firebase STOREPROFILES = new Firebase("https://mtindo.firebaseio.com/");
```

Comment:
I would prefer that you separate "https://mtindo.firebaseio.com/" into strings.xml